cmd/package_managers.go

```diff
@@ -108,6 +146,38 @@ func fetchGitRepositoryFromNPM(packageName string, packageManager pmc.Client) (s
 	return v.Objects[0].Package.Links.Repository, nil
 }
 
+func findGitRepositoryInPYPIResponse(packageName string, response io.Reader) (string, error) {
+	v := &pypiSearchResults{}
+	err := json.NewDecoder(response).Decode(v)
+	if err != nil {
+		return "", sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("failed to parse pypi package json: %v", err))
+	}
+
+	v.Info.ProjectURLs["key_not_used_and_very_unlikely_to_be_present_already"] = v.Info.ProjectURL
+	var validURL string
+	for _, url := range v.Info.ProjectURLs {
+		for _, matcher := range pypiMatchers {
+			repo := matcher(url)
+			if repo == "" {
+				continue
+			}
+			if validURL == "" {
+				validURL = repo
+			} else if validURL != repo {
```

Comment:
Currently, the `strings.ToLower` is in `makeGithubRepo`, so the GitLab pattern wouldn't benefit from it. What do you think about moving the `ToLower` call out of `makeGithubRepo` to here?

```go
for _, url := range v.Info.ProjectURLs {
		url = strings.ToLower(url)
		for _, matcher := range pypiMatchers {
```

Reply:
@spencerschrock sorry just seeing this. I'm happy to move the ToLower to cover gitlab as well. I am not sure if gitlab is case insensitive, but I'll assume you are and make that change now

Reply:
GitLab is case insensitive as well. At least for the owner/repo strings

Reply:
Added to the gitlab parsing bit with a comment...because I'm paranoid about one day there being githubknockoff.com that is case sensitive. Also added a test